packages/build/tests/plugins/tests.js

```diff
@@ -1279,83 +1279,33 @@ test('Does not pin netlify.toml-only plugin versions if there are no site ID', a
   await runWithPluginRunsMock(t, 'pin_config_success', { flags: { siteId: '' } })
 })
 
-// For some tests of the `node-path` flag we have to disable the buildbot_build_plugins_system_node_version feature flag
-// as with it we always rely on the system node version (and is currently set to true by default)
-
-test('Validate --node-path version is supported by our codebase', async (t) => {
-  const nodePath = getNodePath('8.2.0')
-  await runFixture(t, 'node_version_simple', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: false } },
-  })
-})
-
-test('No --node-path version validation is performed if the plugins_system_node_version feature flag is set', async (t) => {
-  const nodePath = getNodePath('8.2.0')
-  await runFixture(t, 'node_version_simple', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: true } },
-  })
-})
-
 test('Validate --node-path unsupported version does not fail when no plugins are used', async (t) => {
   const nodePath = getNodePath('8.2.0')
   await runFixture(t, 'empty', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: false } },
+    flags: { nodePath },
   })
 })
 
 test('Validate --node-path version is supported by the plugin', async (t) => {
-  const nodePath = getNodePath('9.0.0')
+  const nodePath = getNodePath('12.0.0')
   await runFixture(t, 'engines', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: false } },
+    flags: { nodePath },
   })
 })
 
-test('Validate --node-path', async (t) => {
+test('Validate --node-path exists', async (t) => {
   await runFixture(t, 'node_version_simple', {
-    flags: { nodePath: '/doesNotExist', featureFlags: { buildbot_build_plugins_system_node_version: false } },
+    flags: { nodePath: '/doesNotExist' },
   })
 })
 
-test('Provided --node-path version produces a node version warning if the build is being run in buildbot with local plugins and the plugins_system_node_version ff is set to false', async (t) => {
-  const nodePath = getNodePath('9.0.0')
-  await runFixture(t, 'engines', {
-    flags: { nodePath, mode: 'buildbot', featureFlags: { buildbot_build_plugins_system_node_version: false } },
```

Comment:
No longer required as it validated the presence of the warning message mentioned above.